README.md

```diff
@@ -46,9 +50,9 @@ func main() {
 	gocron.Every(1).Monday().Do(task)
 	gocron.Every(1).Thursday().Do(task)
 
-	// function At() take a string like 'hour:min'
-	gocron.Every(1).Day().At("10:30").Do(task)
```

Comment:
I think it is still valuable to have this line here as an example of what you can use.
It isn't always clear what you do need and don't need when you're learning a new language.

Reply:
I agree. It can be retained.